packages/test/test-end-to-end-tests/src/packageVersion.ts

```diff
@@ -6,4 +6,4 @@
  */
 
 export const pkgName = "@fluidframework/test-end-to-end-tests";
-export const pkgVersion = "0.53.0";
+export const pkgVersion = "0.54.0";
```

Comment:
You should not be bumping this in your PRs. Is this a manifest of improperly done bump?

Reply:
I think this happened because of merge. My change was in flight and we released.